Makefile

```diff
@@ -31,12 +31,13 @@ haddock:
 		--fast \
 		>haddock.log 2>&1 \
 		|| ( cat haddock.log; exit 1; )
-	cat haddock.log
-	if grep -B 2 'Module header' haddock.log; then \
+        #remove header warning for Paths_kore
+	sed "N;N;/[^\n]*in 'Paths_kore'\n.*\n.*/d" haddock.log | tee haddock.log.noPaths
```

Comment:
This removes lines containing `in 'Paths_kore'` and the following two lines from the log. Otherwise the subsequent check will complain about the (auto-generated) module not having a doc. header.